### PR TITLE
Submit job form

### PR DESCRIPTION
**Pull request by achiu8**
https://github.com/yujinjcho/rcruitme-front/issues/4

### CODE REVIEW

File: src/SubmitJobForm.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  state = {
+    fields: { ...INITIAL_FIELDS },
+    errors: {}
+  };
+
+  handleChange = field => e =>
+    this.setState({
+      fields: {
+        ...this.state.fields,
+        [field]: e.target.value
```
Comment (yujinjcho) [inline src/SubmitJobForm.js]: how does this bracket work? I assume fields doesn't end up with an array key?

**Comment by yujinjcho** [comment]
can't seem to write job to db. see the following.

screen shot 2019-01-05 at 9 49 18 am

File: src/SubmitJobForm.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    const { fields, errors } = this.state;
+
+    return (
+      <Form onSubmit={this.handleSubmit}>
+        {Object.keys(fields).map(field => (
+          <Form.Group key={field}>
+            <Form.Label>{titleize(field)}</Form.Label>
+            <Form.Control
+              name={field}
+              type="text"
```
Comment (yujinjcho) [inline src/SubmitJobForm.js]: should some of these be textarea? like description?

**Comment by achiu8** [comment]
Hmm.. can you try clearing your cookies in browser?

**Pull request merged**